frame/tips/src/lib.rs

```diff
@@ -408,6 +410,35 @@ decl_module! {
 			Tips::<T>::remove(hash);
 			Self::payout_tip(hash, tip);
 		}
+
+		/// Remove and slash an already-open tip.
+		///
+		/// May only be called from `T::RejectOrigin`.
+		///
+		/// As a result, API will slash the finder and the deposits are lost.
+		///
+		/// Emits `TipSlashed` if successful.
+		///
+		/// # <weight>
+		///   `T` is charged as upper bound given by `ContainsLengthBound`.
+		///   The actual cost depends on the implementation of `T::Tippers`.
+		/// - DbReads: `Tips`,
+		/// - DbWrites: `Reasons`, `Tips`
```

Comment:
```suggestion
		///   `T` is charged as upper bound given by `ContainsLengthBound`.
		///   The actual cost depends on the implementation of `T::Tippers`.
```

@shawntabrizi do you think these length comments are still any useful? I have started removing them in my recent PRs, because now the the bot and your tool that visualises these, all of the numerical details are available. Adding them here is just burden of maintenance. I would keep the weight comments about the high level semantics, e.g. "The weight is assumed as [worse case description] and a refund happens in the case of [other scenario description]".